### DISCUSSION TITLE
Copying maps from Windows to Linux

### DISCUSSION THREAD

**Discussion by Nechron** [ppy/osu#18341] [Q&A] [answered]
Hi! I play lazer on windows. I'd like to switch to linux, but... What folders do I need to copy in order to still have my maps? No matter where I look, I just can't figure it out.

**Comment by fyyrraaa** [comment, marked as answer]
From what I understand, by default on Windows it is located under `%APPDATA%\osu`, and on Linux it's stored under `$HOME/.local/share/osu`. This can be accessed via Settings (General > Open osu! folder).

I think you might be able to get away with copying the entire `osu!` folder to Linux and it'll read the data just fine, but take this with a grain of salt because I haven't tried it myself.

**Reply by Nechron** [reply]
It looks as though the two folders are perfectly identical... Thanks! I will try and copy the files now
